src/styles/VisualEditorRoot.css

```diff
@@ -16,6 +16,7 @@ html::-webkit-scrollbar {
 }
 
 #VisualEditorRoot.root {
-  height: 100%;
+  height: 100vh;
   padding-bottom: 100px;
+  overflow: scroll;
```

Comment:
Not sure this is a good idea... If you're on Macbook, I know scrollbars are generally hidden (I'd recommend any web dev on a MacBook to [do this](https://apple.stackexchange.com/questions/359794/how-do-i-fix-this-annoying-chrome-for-macos-scroll-bar-issue)) but on other systems you might always see them.

Basically, we'd be forcing scrollbars on that element, even if there's no overflow.